l10n_ch_bank_statement_import_postfinance/models/account_bank_statement_line.py

```diff
@@ -31,6 +31,7 @@ class AccountBankStatementLine(models.Model):
     )
     datas = fields.Binary(related='related_file.datas')
     file_ref = fields.Char()
+    svcr_ref = fields.Char()
```

Comment:
This should not be added here, as there is already this field in l10n_ch_import_camt054 module. I think you should check at this module and maybe cut out some functionalities into postfinance import. The module is found on branch pain-sepa. You can copy it on your branch and add it (we will lose commit history but since the module is not published I think we can afford it).